From: Octavio Alvarez <alvarezp@alvarezp.com>
Date: Thu, 30 Oct 2014 01:02:28 -0700
Subject: Don't install OUI and use Debian-provided

We don't ship oui.txt, don't try to install
Debian has an ieee-data package which releases us from shipping
IEEE's oui.txt. We need to tweak GNUmakefile so it doesn't
attempt to install the file, or make install will fail.

Also, the ipv6toolkit.conf file should point to the file
provided by the ieee-data package, located in /var/lib/ieee-data.

Author: Octavio Alvarez <alvarezp@alvarezp.com>
Bug-Debian: http://bugs.debian.org/739202
Forwarded: not-needed
---
 GNUmakefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/GNUmakefile b/GNUmakefile
index e88eb8a..70a2ccf 100644
--- a/GNUmakefile
+++ b/GNUmakefile
@@ -112,7 +112,7 @@ libipv6.o: $(SRCPATH)/libipv6.c $(SRCPATH)/libipv6.h
 data/ipv6toolkit.conf:
 	echo "# SI6 Networks' IPv6 Toolkit Configuration File" > \
            data/ipv6toolkit.conf
-	echo OUI-Database=$(PREFIX)/share/ipv6toolkit/oui.txt >> \
+	echo OUI-Database=/var/lib/ieee-data/oui.txt >> \
            data/ipv6toolkit.conf 
 	echo Ports-Database=$(PREFIX)/share/ipv6toolkit/service-names-port-numbers.csv >> \
            data/ipv6toolkit.conf 
@@ -134,8 +134,8 @@ ifneq ($(OS),SunOS)
 	install -m0644 data/ipv6toolkit.conf $(ETCPATH)
 
 	# Install the IEEE OUI database
-	install -m0755 -d $(DATAPATH)
-	install -m0644 data/oui.txt $(DATAPATH)
+	#install -m0755 -d $(DATAPATH)
+	#install -m0644 data/oui.txt $(DATAPATH)
 
 	# Install the port numbers database
 	install -m0644 data/service-names-port-numbers.csv $(DATAPATH)
